src/components/indicators.js

```diff
@@ -165,24 +183,24 @@ export const loadingIndicatorCSS = ({
   verticalAlign: 'middle',
 });
 
-type DotProps = { color: string, delay: number, offset: boolean, emotion: any };
-const LoadingDot = ({ color, delay, offset, emotion }: DotProps) => (
-  <span
-    className={emotion.css({
-      animationDuration: '1s',
-      animationDelay: `${delay}ms`,
-      animationIterationCount: 'infinite',
-      animationName: keyframesName,
-      animationTimingFunction: 'ease-in-out',
-      backgroundColor: color,
-      borderRadius: '1em',
-      display: 'inline-block',
-      marginLeft: offset ? '1em' : null,
-      height: '1em',
-      verticalAlign: 'top',
-      width: '1em',
-    })}
-  />
+type DotProps = { color: string, delay: number, offset: boolean };
+const LoadingDot = ({ color, delay, offset }: DotProps) => (
```

Comment:
@mitchellhamilton same as with the svg example above, technically not necessary to use <ClassNames/> here, but noticed that nonces were not being added, and the keyframes`` call above was not working.